spec/generator/metagem/engine_models_generator_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+require_relative '../../../lib/generators/rails/model_override.rb'
+require_relative '../../../lib/generators/metagem/engine_model/engine_model_generator.rb'
+
+describe Rails::Generators::ModelGenerator, type: :generator do
+  let(:model_name) { "feedback/session" }
+  let(:spec_file) { file("spec/engines/feedback/models/feedback/session_spec.rb") }
+  let(:model_file) { file("engines/feedback/app/models/feedback/session.rb") }
+  let(:model_namespcae) { file("app/models/feedback.rb") }
+  let(:migration) { Dir["#{Rails.root}/engines/feedback/db/migrate/*"].max }
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Rails/FilePath: Prefer Rails.root.join('path/to').